### PR TITLE
Add Dense Prediction Transformer implementation

### PR DESCRIPTION
**Pull request by abhi-glitchhg**

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -391,3 +391,51 @@ def test_cct():
...
+
+
+def test_dpt():
+    img = torch.randn(4, 3, 384, 384)
+    model = MODEL_REGISTRY.get("DPTDepth")(
+        "vitb16",
+        enable_attention_hooks=True,
+    )
+    del model
+    model = MODEL_REGISTRY.get("DPTDepth")("vitl16")
```
Comment (aditya-agrawal-30502) [inline tests/test_models.py:403]:
```suggestion

    model = MODEL_REGISTRY.get("DPTDepth")("vitl16")
```

File: tests/test_models.py [hunk 2]
```diff
@@ -391,3 +390,74 @@ def test_cct():
...
+    del model
+
+
+def test_dpt():
+    img = torch.randn(4, 3, 384, 384)
+    model = MODEL_REGISTRY.get("DPTDepth")(
+        "vitb16",
+        enable_attention_hooks=True,
+    )
+    del model
```
Comment (aditya-agrawal-30502) [inline tests/test_models.py:424]:
```suggestion
    del model
    
```

Comment (abhi-glitchhg) [reply tests/test_models.py:424]: @aditya-agrawal-30502  
Umm i did not get you

**Comment by abhi-glitchhg** [comment]
I had to copy few files from  #56 and #57 to resolve conflicts

**Pull request merged**